plugins/mcreator-localization/help/default/block/light_opacity.md

```diff
@@ -1,3 +1,2 @@
-This property makes the block allow or not allow the light to pass through, 
-126 makes it semi-transparent (only in some Minecraft versions), 255 will block all light (default value), 
-and 0 will allow all light through.
\ No newline at end of file
+This property determines how much the light level decreases when going through this block. For example, a value of 15

```

Comment:
Could you change the French help too please? Copy/paste the following :

"Cette propriété détermine de combien le niveau de lumière diminue en traversant ce bloc. Par exemple, une valeur de 15 permet au bloc de bloquer toute la lumière, tandis qu'une valeur de 0 rendra le bloc transparent."